android/src/main/java/io/ably/flutter/plugin/AblyMessageCodec.java

```diff
@@ -136,6 +136,20 @@ private void WriteJsonElement(ByteArrayOutputStream stream, JsonElement value){
         }
     }
 
+    /**
+     * Converts Map to JsonObject, ArrayList to JsonArray and
+     * returns null if these 2 types are a no-match
+     * */
```

Comment:
What's with the space in the middle here? I would have expected:

```suggestion
     **/
```

Reply:
Addressed all java code formatting

052467d338e6fd8a37e3474c03b355e0ec1b8fdc